### PR TITLE
chore: Nuxt 3 pre-release prep

### PR DESCRIPTION
**Pull request by codebender828**
Thanks @pi0 for the review! 

I made a few changes as per your review! Thank you so much.

- [x] Remove temp repo https://github.com/nuxt-community/emotion-module-nuxt3 (This was archived. Might need higher access to delete the repo completely :))
- [x]  Update nuxt/modules DB (https://github.com/nuxt/modules/issues/556)
- [x] Update readme note
- [x] Small review comments in https://github.com/nuxt-modules/emotion/pull/21 i might come with more tips. we can switch to pnpm for example too

Closes nuxt-modules/emotion#22

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Install dependencies
         if: steps.cache.outputs.cache-hit != 'true'
-        run: yarn
+        run: yarn && yarn dev:prepare
```
Comment (pi0) [inline .github/workflows/ci.yml:42]: This should be pnpm now no? (with `corepack enable` step before)
```suggestion
        run: pnpm enable && pnpm install && pnpm dev:prepare
```

Comment (codebender828) [reply .github/workflows/ci.yml]: Thanks! Updated this script.

**Comment by pi0** [comment]
Feel free to merge and release when ready 🚀

**Comment by codebender828** [comment]
Woops 😄 Looks like I need write access in order to merge.
CleanShot 2023-01-22 at 04 28 17@2x

**Pull request merged**